manifests/s/ShiftUp/NIKKE/0.0.6.178/ShiftUp.NIKKE.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ShiftUp.NIKKE

```

Comment:
Where does `ShiftUp` come from? The Installer URL and Publisher don't seem to match with `ShiftUp`

Reply:
if you go to https://nikke-en.com/ and scroll all the way down, there it is

Reply:
> ©Proxima Beta Pte. Limited ©SHIFT UP CORP.